src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
you don't have to set them directly, let the content decide, what's the total width / height of your block